plugins/in_syslog/syslog_server.c

```diff
@@ -64,6 +65,12 @@ static int syslog_server_unix_create(struct flb_syslog *ctx)
         return -1;
     }
 
+    if (chmod(address.sun_path, ctx->unix_perm)) {
+        flb_errno();
+        close(fd);
```

Comment:
can you append here a descriptive additional error, e.g: flb_error("[in_syslog] error setting permissions 0666 on path..)

Reply:
Thank you. b52523c1 adds an error message there.
The message looks like the following:

> [2019/09/05 14:08:59] [error] [in_syslog] cannot set permission on '/tmp/fluentbit.sock' to 0666

I also changed `perm_unix` to unsinged to follow the convention.